internal/pkg/fetch/fetch.go

```diff
@@ -51,6 +51,16 @@ func Fetch(sbomURL string, opts *options.FetchOptions) error {
 		return fmt.Errorf("error adding document: %w", err)
 	}
 
+	if opts.Alias != "" {
+		if err := backend.SetAlias(doc.GetMetadata().GetId(), opts.Alias, false); err != nil {
+			return fmt.Errorf("%w", err)
+		}
+	}
+
+	if err := backend.AddAnnotations(doc.GetMetadata().GetId(), db.TagAnnotation, opts.Tags...); err != nil {
+		return fmt.Errorf("failed to set tags: %w", err)
```

Comment:
I just realized we probably don't want to return here on error. I'll change this to use `errors.Join` to avoid alias errors from canceling the `fetchExternalReferences` action and leaving the document incomplete.

Reply:
Per Jonathan's suggestion, I changed it to log a warning instead.